src/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 find_package(Boost REQUIRED filesystem system serialization)
 find_package(SCIP)
 
+rock_find_qt4(OPTIONAL QtGui QtSvg)
+rock_find_qt5(OPTIONAL Gui Svg Widgets PrintSupport)
```

Comment:
For Qt4, QtGui(and QtCore, QtOpenGL) and for Qt5, Gui, Widgets(and Core, OpenGL) are added to components by default. This does no harm, though.